src/demolition_spider.py

```diff
@@ -97,4 +98,8 @@ def extractRecords(self, response, records_rows):
         # make sure that we actually extracted some records and it was a valid table
         if len(records) == 0:
             raise Exception("No records found in table")
+        else:
```

Comment:
Rather than an email per record, the goal is to send an alert that there are new records. Ideally, the email should contain:
1) a link to the report interface (the same page we scrape)
2) a table or list of discovered records

Reply:
Alright turned it into a table and link to the report interface now!